src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -859,20 +859,21 @@ private String getContributionOptionsColor(int points) {
      * @return points in text form "Equal Share..."
      */
     public static String convertToEqualShareFormat(int i) {
-        if (i > 100)
+        if (i > 100) {
             return "Equal share + " + (i - 100) + "%"; // Do more
-        else if (i == 100)
+        } else if (i == 100) {
             return "Equal share"; // Do same
-        else if (i > 0)
+        } else if (i > 0) {
             return "Equal share - " + (100 - i) + "%"; // Do less
-        else if (i == 0)
+        } else if (i == 0) {
             return "0%"; // Do none
-        else if (i == Const.POINTS_NOT_SURE)
+        } else if (i == Const.POINTS_NOT_SURE) {
             return "Not Sure";
-        else if (i == Const.POINTS_NOT_SUBMITTED)
+        } else if (i == Const.POINTS_NOT_SUBMITTED) {
             return "";
-        else
+        } else {
             return "";
+        }
```

Comment:
The `i == Const.POINTS_NOT_SUBMITTED` might not be necessary because the return value is the same as the `else` clause.